### DISCUSSION TITLE
Existing module doesn't load input fields within blade file

### DISCUSSION THREAD

**Discussion by adam-jones-net** [area17/twill#2352] [Q&A]
Hi all, I've no idea how it happened but just one module in my Twill 2.13 install has stopped rendering its blade template correctly.   On my local dev it works.  There is a wysiwyg field and then some image fields and they work fine like this:

![image](https://github.com/area17/twill/assets/3810108/d7f7b31e-de8a-4f12-ba7b-4308611d935b)

But on our live server all we get is this:

![image](https://github.com/area17/twill/assets/3810108/4f8992f4-573e-4964-bf7e-0c911e67937f)

If I inspect the served html on the problem server I can see that the fieldset__content div is empty whereas on the working setup it contains many child objects and critically the input fields.   So it's not the case that css/js has broken on the page, but that twill didn't even render the input fields into the html.

![image](https://github.com/area17/twill/assets/3810108/8fc79fd3-816e-4f1c-9b17-70df0ad0d2a0)

I've cleared caches, rebuilt blocks etc but no joy.   How can I troubleshoot this further ?

**Comment by ifox** [comment]
Hi @adam-jones-net, can you share the blade form code? What about new records? Do they have the same problem? If not, I would try to import the record where you are reproducing this on your local db to try to reproduce locally.

**Reply by adam-jones-net** [reply]
thanks for the reply @ifox .   Updating the existing record or saving a new record both resultin an error,  so maybe that is part of it?  Or maybe it's just the result of the render problem?


>     "message": "Checkinpage",
>     "exception": "A17\\Twill\\Exceptions\\NoCapsuleFoundException",
>     "file": "/home/customer/www/test.com/vendor/area17/twill/src/TwillCapsules.php",
>     "line": 75,
>     "trace": [
>         {
>             "file": "/home/customer/www/test.com/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php",
>             "line": 338,
>             "function": "getCapsuleForModel",
>             "class": "A17\\Twill\\TwillCapsules",
>             "type": "->"
>         },

Blade code is below:

```
@extends('twill::layouts.form', [
    'additionalFieldsets' => [
        ['fieldset' => 'media', 'label' => 'Media']
    ]
])

@section('contentFields')
    @formField('wysiwyg', [
        'name' => 'description',
        'type' => 'textarea',
        'label' => 'Check-in instructions',
        'maxlength' => 5000
    ])
@stop



@section('fieldsets')
    <a17-fieldset title="Media for this page" id="media">
        @formField('medias', [
            'name' => 'checkin_photos',
            'label' => 'Check-in Photos',
            'required' => false,
            'max' => 5,
            'note' => 'Add up to 5 photos to compliment the instructions you have written'
        ])

        @formField('files', [
            'name' => 'instructional_video',
            'label' => 'Instructional Video',
            'note' => 'Note that you can only upload one video per check-in page',
            'max' => 1,
        ])
    </a17-fieldset>
@stop
```

**Reply by ifox** [reply]
I see `Checkinpage` in the error. Can you confirm that the capsule folder name and model are using this casing? This looks like a potential case sensitivity issue where your file or folder name is actually `CheckinPage`. Locally you're most likely using a case insensitive filesystem and not seeing the error.

**Reply by adam-jones-net** [reply]
@ifox I don't have a capsule but the model uses Checkinpage.php.  The holder of the blade template however wasn't with correct casing, so that was it.     This site uses twill 2 still but out of interest, have things like this been made any clearer in the errors in Twill 3 ?
